packages/kbn-dev-utils/src/babel.ts

```diff
@@ -46,3 +46,14 @@ export async function transformFileWithBabel(file: File) {
   file.extname = '.js';
   transformedFiles.add(file);
 }
+
+/**
+ * Synchronized regex list of files that use `styled-components`.
+ * Used by `kbn-babel-preset` and `elastic-eslint-config-kibana`.
+ */
+export const USES_STYLED_COMPONENTS = [
+  /packages[\/\\]kbn-ui-shared-deps[\/\\]/,
+  /src[\/\\]plugins[\/\\](data|kibana_react)[\/\\]/,
+  /x-pack[\/\\]plugins[\/\\](apm|beats_management|cases|fleet|infra|lists|observability|osquery|security_solution|timelines|uptime)[\/\\]/,
+  /x-pack[\/\\]test[\/\\]plugin_functional[\/\\]plugins[\/\\]resolver_test[\/\\]/,
```

Comment:
Have you created a meta issue to ping codeowners of these plugins to migrate from `styled components`?

Reply:
I have not. Thanks for the reminder